_config.yml

```diff
@@ -89,7 +89,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Notre partenaire pour les phases de test sur le terrain, et notre intermédiaire avec les pompiers."
+        content: "Acteur de l'innovation pour les secours, c'est notre point de relai avec les pompiers."
+      sponsor3:
+        title: "SDIS Ardeche"
+        url: "https://www.sdis07.fr/"
+        logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+        content: "Notre partenaire historique, du recueil des besoins aux phases de test sur le terrain."
```

Comment:
Not sure this is the right way to put it: Atraksis was supporting us way before

Perhaps "Notre partenaire pour le recueil des besoins et l'expérimentation terrain."

Reply:
Well, I do not fully agree, we've met Atraksis after and already have engaged discussions with Ardèche. But, I see your point, at we could say both are there from the start so your suggestion suits me :)